apps/entry/widgets/date_range_widget.py

```diff
@@ -35,9 +36,16 @@ def update_attribute(widget, data, widget_data):
 
         'export_data': {
             'data': {
-                'excel': {
+                'version': 1,
+                'common': {
                     'values': [from_date, to_date],
+                    'widget_id': WIDGET_ID,
+                    'widget_key': widget.key,
+                },
+                'excel': {
                 },
+                'report': {
+                }
```

Comment:
If we are using `.get('excel', {})` in export, then we will not need this. What do you think?

Reply:
Thought about that. But left it as so such that next viewer will not have to go through the code to know the schema of the json data.